src/ai/backend/client/output/console.py

```diff
@@ -18,7 +20,7 @@ class NoItems(Exception):
 class ConsoleOutputHandler(BaseOutputHandler):
     def print_item(
         self,
-        item: Mapping[str, Any] | None,
+        item: Optional[Mapping[str, Any]],
```

Comment:
We can also utilize `_Item` type alias here.
```suggestion
        item: Optional[_Item],
```

Reply:
I found there are `Mapping` and `MutableMapping`.
In our context, I think `Mapping` is more suitable for the item. An abstract class `BaseOutputHandler` which declared the item as `Mapping`. A consuming function `tabulate_items` declared the item as `MutableMapping` but there's no mutation and it just iterates the items.

Reply:
Please review the following commit.